### PR TITLE
Re-activate and update Zillow.com related rulesets

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Zillow.com.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		Certificate mismatched:
+		- investors.zillow.com
+
+		Status code mismatch:
+		- mortgageapi.zillow.com
+
+		Mixed content blocking (MCB) triggered:
+     		- info.zillow.com
+-->
+<ruleset name="Zillow.com">
```
Comment (Bisaloo) [inline src/chrome/content/rules/Zillow.com.xml]: Shouldn't this be `(partial)`?

File: src/chrome/content/rules/Zillow.com.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	<target host="zillow.com" />
+	<target host="www.zillow.com" />
+	<target host="premieragent.zillow.com" />
+	<target host="photos1.zillow.com" />
+	<target host="photos2.zillow.com" />
+	<target host="photos3.zillow.com" />
+
+	<securecookie host=".+" name=".+" />
+
```
Comment (Bisaloo) [inline src/chrome/content/rules/Zillow.com.xml]: You have trailing whitespaces at multiple places in this file

File: src/chrome/content/rules/Zillow.com.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+		- engineering.zillow.com
+
+		Certificate mismatched:
+		- investors.zillow.com
+
+		Status code mismatch:
+		- mortgageapi.zillow.com
+
+		Mixed content blocking (MCB) triggered:
+     		- info.zillow.com
```
Comment (Bisaloo) [inline src/chrome/content/rules/Zillow.com.xml]: Could you also provide an example of an URL where you get MCB please? I simply get a (302->)404 on https://info.zillow.com/

File: src/chrome/content/rules/Zillow.com.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+-->
+<ruleset name="Zillow.com">
+	<target host="zillow.com" />
+	<target host="www.zillow.com" />
+	<target host="premieragent.zillow.com" />
+	<target host="photos1.zillow.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Zillow.com.xml]: Could you add a test url please? E.g. http://photos1.zillow.com/i_g/ISatx8mbaxt0v7.jpg

File: src/chrome/content/rules/Zillowstatic.com.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="Zillowstatic.com">
+	<target host="zillowstatic.com" />
+	<target host="www.zillowstatic.com" />
+	<target host="photos1.zillowstatic.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Zillowstatic.com.xml]: Could you add a test url please? http://photos1.zillowstatic.com/i_g/ISatx8mbaxt0v7.jpg

File: src/chrome/content/rules/Zillowstatic.com.xml [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+	<target host="photos2.zillowstatic.com" />
+	<target host="photos3.zillowstatic.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Zillowstatic.com.xml]: Please add `wp.zillowstatic.com` with test url http://wp.zillowstatic.com/1/vert-one-resized-169851.jpg

File: src/chrome/content/rules/Zillowstatic.com.xml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="Zillowstatic.com">
+	<target host="zillowstatic.com" />
+	<target host="www.zillowstatic.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Zillowstatic.com.xml]: `photos.zillowstatic.com`

Comment (Bisaloo) [inline src/chrome/content/rules/Zillowstatic.com.xml]: \iss.zillowstatic.com` with http://iss.zillowstatic.com/image/tropical-bar-with-outdoor-kitchen-i_g-ISxvusi0aodvjl0000000000-GqZQt.jpg

**Comment by cschanaj** [comment]
@Bisaloo Updated the rulesets accordingly, but I cannot see the trailing white spaces with `cat -e`, would you kindly confirm?

**Comment by Bisaloo** [comment]
> but I cannot see the trailing white spaces with cat -e, would you kindly confirm?

You're right, I was fooled by GitHub UI

**Comment by Bisaloo** [comment]
@cschanaj, I think you forgot two comments:

> Could you add a test url please? http://photos1.zillowstatic.com/i_g/ISatx8mbaxt0v7.jpg

> Please add wp.zillowstatic.com with test url http://wp.zillowstatic.com/1/vert-one-resized-169851.jpg

I marked the rest as resolved

File: src/chrome/content/rules/Zillowstatic.com.xml (hunk 6 above)
Comment (Bisaloo) [reply src/chrome/content/rules/Zillowstatic.com.xml]: Oh, you actually added it but to the wrong file

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**